Projects/Addons/Sources/Views/AddonProcessingScreen.swift

```diff
@@ -0,0 +1,48 @@
+import SwiftUI
+import hCore
+import hCoreUI
+
+struct AddonProcessingScreen: View {
+    @ObservedObject var vm: ChangeAddonViewModel
+    @EnvironmentObject var addonNavigationVm: ChangeAddonNavigationViewModel
+
+    var body: some View {
+        ProcessingStateView(
+            loadingViewText: L10n.tierFlowCommitProcessingLoadingTitle,
+            successViewTitle: L10n.addonFlowSuccessTitle,
+            successViewBody: L10n.addonFlowSuccessSubtitle,
+            successViewButtonAction: {
+                addonNavigationVm.router.dismiss()
+            },
+            state: $vm.viewState
```

Comment:
Not sure if it is smart to use the same viewState for getting tiers and submitting tiers.
Can we have different states for getting and submitting addon inside ChangeAddonViewModel ?

Reply:
Yes that's a good idea